ospf6d/ospf6_lsdb.c

```diff
@@ -92,6 +92,15 @@ static void _lsdb_count_assert(struct ospf6_lsdb *lsdb)
 #define ospf6_lsdb_count_assert(t) ((void) 0)
 #endif /*DEBUG*/
 
+static inline void ospf6_lsdb_stats_update(struct ospf6_lsa *lsa,
+					   struct ospf6_lsdb *lsdb, int count)
+{
+	uint16_t stat = ntohs(lsa->header->type) & OSPF6_LSTYPE_FCODE_MASK;
+	if ((stat >= OSPF6_LSTYPE_SIZE) || (stat == 0x0a))
```

Comment:
I considered the empty value when I wrote the sample code. I don't think you should special case it here. This is the type of thing that doesn't get updated when that type gets defined later on and then we have a bug in the code. I think its OK to count the gap "unknowns"  separate from the "out of range" values.

Reply:
Done.